test/integration/aens.ts

```diff
@@ -99,7 +100,7 @@ describe('Aens', function () {
 
```

Comment:
**line 98:** throws an `Returning an awaited promise is required in this context.` error. Should be `retrun await...`